src/main/java/io/wcm/caconfig/extensions/persistence/impl/ToolsConfigPagePersistenceStrategy.java

```diff
@@ -98,16 +98,20 @@ public class ToolsConfigPagePersistenceStrategy implements ConfigurationPersiste
         description = "Priority of persistence strategy (higher = higher priority).")
     int service_ranking() default 2000;
 
+    @AttributeDefinition(name = "Relative config path",
+        description = "Relative path to the configuration page content.",
+        defaultValue = "/tools/config/jcr:content")
```

Comment:
it should not be required to define the default value twice - should be sufficient to do it in the annotation class